src/content/docs/reference/Uniforms/Matrices/gbufferModelViewInverse.mdx

```diff
@@ -4,9 +4,8 @@ description: gbufferModelViewInverse
 sidebar:
   label: gbufferModelViewInverse
   order: 1
-  badge:
-    text: Unfinished
-    variant: caution
 ---
 
-### `uniform ;`
\ No newline at end of file
+### `uniform mat4 gbufferModelViewInverse;`
+
+Equal to `inverse(gbufferModelView)`, but calculated on the CPU for improved performance since the `inverse()` function is very slow.
```

Comment:
You should probably mention that this is equivalent to `gl_ModelViewMatrix` (since the compatibility profile is preferred over the core profile for Iris)

Reply:
Right👍 Are there any more compatibility profile equivalents I should add?

Reply:
accidentally answered this in an earlier comment